### ISSUE TITLE
Listview iOS: Expose Cell Selection\focus colors

### ISSUE THREAD

**Issue by sitefinitysteve** [NativeScript/NativeScript#1791]
### Did you verify this is a real problem by searching [Stack Overflow]
Yes

### Tell us about the problem
When you click a Listview UITableViewCell it highlights grey.  Could we please expose the ability to change this? (I think here) https://developer.apple.com/library/ios/documentation/UIKit/Reference/UITableViewCell_Class/#//apple_ref/c/tdef/UITableViewCellSelectionStyle

It's causing styling issues here...I have a label with an iconfont that has a circle styled around it (with css).  When the list item is clicked, the circle goes away... almost like it loses it's padding and border-radius or something.

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
All

### Please tell us how to recreate the issue in as much detail as possible.
Make a listview, click a listitem

**Comment by manijak** [comment]
I agree that this should be built inn, so that we have an easy way of overriding it. 

Here is a temporary workaoround, add to the 'itemLoading' event of ListView:
```js
exports.listViewItemLoading = function(args) {
    if (frameModule.topmost().ios) {
        var cell = args.ios;
        cell.selectionStyle = UITableViewCellSelectionStyle.UITableViewCellSelectionStyleNone;
    }
}
```

**Comment by sitefinitysteve** [comment]
Thanks for the workaround!  ...should be core or settable somehow, or even set to "None" by default

**Comment by RoyiNamir** [comment]
In angular it would be `(itemLoading)=...`

For angular folks
